meta-multimedia/recipes-multimedia/clight/clightd_git.bb

```diff
@@ -0,0 +1,39 @@
+SUMMARY = "D-Bus service providing screen management and ambient light detection"
+HOMEPAGE = "https://github.com/FedeDP/clightd"
+SECTION = "utils"
+LICENSE = "GPL-3.0-only"
+LIC_FILES_CHKSUM = "file://COPYING;md5=d32239bcb673463ab874e80d47fae504"
+
+SRC_URI = "git://github.com/FedeDP/${BPN};protocol=https;branch=master"
+SRCREV = "ffb5213f1a9b5b045396832a4c8c93fd45c55167"
+S = "${WORKDIR}/git"
+
+
+inherit cmake pkgconfig
+
+DEPENDS += "\
+    udev \
+    polkit \
```

Comment:
`polkit` dependency should be controlled with

```bitbake
inherit features_check
REQUIRED_DISTRO_FEATURES = "polkit"
```

ideally if possible, it should be a PACKAGECONFIG which is disabled by default and enabled only if polkit it in distro features.